rasterio/_warp.pyx

```diff
@@ -165,6 +165,7 @@ def _reproject(
         dst_crs=None,
         dst_nodata=None,
         resampling=Resampling.nearest,
+        multiple=False,
```

Comment:
@dnomadb what about `init_dest_nodata=True` (or better name)?  Warp doesn't need to know that there are multiple files involved, only that it should or should not initialize the destination with NODATA values.